README.md

```diff
@@ -176,6 +176,19 @@ Pipeline is defined in `.buildkite/pipeline.yml`
 * Tests: Uses the official [python:3.6](https://hub.docker.com/_/python/) image.
 * Style: Uses [black](https://hub.docker.com/r/stellargraph/black/) from the `stellargraph` docker hub organisation.
 
+## Citing
+If you use any part of this library in your research, please cite it using the following BibTex entry
+```latex
+@misc{stellargraph,
+  author = {Data61},
+  title = {Stellar Graph Machine Learning Library},
```

Comment:
We should use StellarGraph as one word, it will help our search rankings

Reply:
Also the main heading on L3 should also say StellarGraph as one word...

Reply:
fixed